tests/test_env.py

```diff
@@ -0,0 +1,61 @@
+"""Test the env module."""
+import os
+from pathlib import Path
+
+import pytest
+from pyper import env
+
+_TEST_DOTENV_PATH = Path(__file__).parent / "data" / ".env"
```

Comment:
You should also test a file with Windows line endings.

Reply:
Since I learned that the `open` function takes care of the newline conventions of different operating systems, I think this is not necessary anymore. See [our conversation](https://github.com/leoschleier/pyper/pull/3/files#r1335143175) above.